components/esp_modem/examples/simple_cmux_client/pytest_cmux.py

```diff
@@ -13,5 +13,5 @@ def test_cmux_connection(dut):
     # Check the sequence of connecting, publishing, disconnecting
     dut.expect('Modem has correctly entered multiplexed')
     # Check for MQTT connection and the data event
-    dut.expect('TOPIC: /topic/esp-modem')
-    dut.expect('DATA: Hello modem')
+    dut.expect('TOPIC: /ci/esp-modem/pppos-client')
```

Comment:
IIRC, we can get the option set in Kconfig.

Reply:
used `dut.app.sdkconfig.get(()` to get these config options